### PR TITLE
feat(starters): Add .prettierrc to starters

### PR DESCRIPTION
**Pull request by thefrontendwizard**
Added `.prettierrc` to all starters following the same rules on the repo (#11778).

### CODE REVIEW

File: starters/blog/package.json [hunk 1]
```diff
@@ -50,6 +53,7 @@
     "develop": "gatsby develop",
     "start": "npm run develop",
     "serve": "gatsby serve",
-    "test": "echo \"Write tests! -> https://gatsby.app/unit-testing\""
+    "test": "echo \"Write tests! -> https://gatsby.app/unit-testing\"",
+    "format": "prettier --write src/**/*.{js,jsx}"
```
Comment (DSchau) [inline starters/blog/package.json]: Nit, and I'm sorry already--but can we move this up after develop? (for all starters)

It's nice to keep these consistent and well-structured.

Once done we can get this merged!

Comment (thefrontendwizard) [reply starters/blog/package.json]: Sure. No problem.

Comment (thefrontendwizard) [reply starters/blog/package.json]: Fixed it.

**Comment by thefrontendwizard** [comment]
There's some weird errors going on CI...

**Comment by sidharthachatterjee** [comment]
@thefrontendwizard Just rebased on master and force pushed to get the (unrelated) CI issue resolved

**Comment by thefrontendwizard** [comment]
> @thefrontendwizard Looks good overall! Small thing — could you add `.prettierrc` to the file trees (and the description in the list) for all three READMEs?

done 👍

File: starters/hello-world/package.json [hunk 2]
```diff
@@ -16,6 +17,12 @@
     "react": "^16.8.3",
     "react-dom": "^16.8.3"
   },
+  "devDependencies": {
+    "prettier": "^1.16.4"
+  },
+  "devDependencies": {
```
Comment (DSchau) [inline starters/hello-world/package.json]: Whoops!

Comment (thefrontendwizard) [reply starters/hello-world/package.json]: Whoops indeed, fixed it 😅

**Comment by DSchau** [comment]
@thefrontendwizard I'll wait for the CI tests to pass, then merge. Thanks!

**Pull request merged**